change/@fluentui-react-charting-effaa597-2236-4e8c-b8f3-91a2422c9f44.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 1 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27415&commitId=214f2bd1fc523caec8e0fc0f57b8159a9ba49c4f&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1925 screenshots unchanged, 0 screenshots with different dimensions and 1 screenshots with visible difference.


  <details><summary><b>unknown</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.size+image+badge.normal.chromium.png|1| Changed|

</details>


<div id="vrtCommentfluentuiv9"/>